base/src/main/java/org/geysermc/api/GeyserApiBase.java

```diff
@@ -116,15 +117,30 @@ public interface GeyserApiBase {
 
     /**
      * Returns the major API version. Bumped whenever a significant breaking change or feature addition is added.
+     *
+     * @deprecated in favor of {@link #baseApiVersion()}.
      */
+    @Deprecated
     default int majorApiVersion() {
-        return 1;
+        return baseApiVersion().major();
     }
 
     /**
      * Returns the minor API version. May be bumped for new API additions.
+     *
+     * @deprecated in favor of {@link #baseApiVersion()}.
      */
+    @Deprecated
     default int minorApiVersion() {
-        return 0;
+        return baseApiVersion().minor();
+    }
+
+    /**
+     * Returns the Base API version.
+     *
+     * @return {@link ApiVersion}
+     */
+    default ApiVersion baseApiVersion() {
```

Comment:
Should we call it `baseApiVersion` or `apiVersion`?

Reply:
i would keep it ```baseApiVersion()```, since it is the ApiVersion of the Base API. Similarly, the Geyser API would have a ```geyserApiVersion()``` - this would avoid confusion imo